### PR TITLE
Sighting model improvements

### PR DESCRIPTION
**Pull request by gbuisson**
- HighMedLow should have a new value, Info - which indicates it's purely informational.

- add `internal` at the root of the model `s/Bool`

- add `severity` at the root of the model, using HighMedLowInfo Vocabulary

- add `resolution` at the root of the model `s/Str` or `s/enum`, this field indicates if the sensor that is reporting the `Sighting` already took action on it, for instance a Firewall blocking the IP.
  - detected
  - blocked
  - allowed
  - contained

- add `observed_time` to `Sighting/Target`, make it required
- make `Sighting/Target` a collection for th Sighting summary feature

### CODE REVIEW

**Comment by msprunck** [comment]
Could you also convert the target field to a collection?

File: src/ctim/schemas/sighting.cljc [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
-            [ctim.schemas.relationship :as rel]
-            [ctim.schemas.vocabularies :as v]
-            #?(:clj  [flanders.core :as f :refer [def-entity-type def-eq def-map-type]]
-               :cljs [flanders.core :as f :refer-macros [def-entity-type def-eq def-map-type]])))
+  #?@
+  (:clj
+   [(:require
+     [ctim.schemas.common :as c]
+     [ctim.schemas.relationship :as rel]
+     [ctim.schemas.vocabularies :as v]
```
Comment (msprunck) [inline src/ctim/schemas/sighting.cljc]: Why do you put these namespaces in the the reader conditionals part? It's common to clj and cljs

Comment (gbuisson) [reply src/ctim/schemas/sighting.cljc]: cider and clj-format do that when you automatically reformat the ns,
I wonder if I prefer manually maintain the conditionals or let the tools do it.

**Comment by msprunck** [comment]
Don't forget to update `ctim-schema-version` when you will release the next version of CTIM

**Pull request merged**